packages/browser/test/e2e/playwright.config.js

```diff
@@ -0,0 +1,13 @@
+const config = {
```

Comment:
Can we type this as per [testConfig.globalSetup](https://playwright.dev/docs/api/class-testconfig#test-config-global-setup)?

_Same question can apply to all other files_

Reply:
The new structure will change the general config a bit. I'll try to type as much code as possible in the new structure 🤞